### PR TITLE
changed user's primaryEmails to be consistent with whatever email's we'v...

### PR DESCRIPTION
**Pull request by ahsu1230**
...e always been using

@andrewconner @leogrim @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 1]
```diff
@@ -891,14 +891,22 @@ class AdminUserController @Inject() (
...
+
+      def pickAnEmail(userId: Id[User]) = {
+        /*
+        val all = emailRepo.getAllByUser(user.id.get)
+        all.filter(_.verified).sortBy(e => (e.lastVerificationSent, e.verifiedAt)).lastOption match {
+          case Some(verifiedEmail) => Some(verifiedEmail.address)
+          case _ => None // no verified emails to choose from, so still None
+        }
+        */
+        Some(emailRepo.getByUser(userId))
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala]: Here's the 

``` scala
def getByUser(userId: Id[User])(implicit session: RSession): EmailAddress = {
    val user = userRepo.get(userId)
    user.primaryEmail getOrElse {
      val all = getAllByUser(userId)
      all.find(_.verified) match {
        case Some(verifiedEmail) => verifiedEmail.address
        case None => all.headOption.getOrElse(throw new Exception(s"no emails for user $userId")).address
      }
    }
  }
```

**Comment by ahsu1230** [comment]
checked by @stkem

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 2]
```diff
@@ -890,19 +890,34 @@ class AdminUserController @Inject() (
...
+    db.readWrite { implicit s =>
+
+      def pickAnEmail(user: User) = {
+        /*
+        val all = emailRepo.getAllByUser(user.id.get)
+        all.filter(_.verified).sortBy(e => (e.lastVerificationSent, e.verifiedAt)).lastOption match {
+          case Some(verifiedEmail) => Some(verifiedEmail.address)
+          case _ => None // no verified emails to choose from, so still None
+        }
+        */
```
Comment (joshm1) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:903]: how come this is commented out?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:903]: We found out it runs faster when it's commented out.

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 3]
```diff
@@ -476,7 +476,7 @@ GET     /admin/user/:id/mixpanel/reset    @com.keepit.controllers.admin.AdminUse
 POST    /admin/user/:id/uservalue   @com.keepit.controllers.admin.AdminUserController.userValue(id: Id[User])
 POST    /admin/user/:id/username   @com.keepit.controllers.admin.AdminUserController.setUsername(id: Id[User], username: String, overrideRestrictions: Boolean ?= false)
 DELETE  /admin/user/:id/username    @com.keepit.controllers.admin.AdminUserController.removeUsername(id: Id[User])
-POST    /admin/user/selectPrimaryEmails @com.keepit.controllers.admin.AdminUserController.selectPrimaryEmails()
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:479]: For migrations, I do prefer `POST`s. Semantically correct, and less dangerous someone will accidentally call it.

Comment (stkem) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:479]: I have to take the blame for this one. I suggested using a get to make it easier to call (and it's non destructive) :-/
You have a good point though.

**Pull request merged**